src/main/java/racingcar/Car.java

```diff
@@ -10,4 +12,16 @@ public Car(String name) {
     }
 
     // 추가 기능 구현
+    public void moveForward() {
+        int move = Randoms.pickNumberInRange(0,9);
+        if(move >= 4) {
```

Comment:
moveForward에 사용된 숫자 4는 매직 넘버로 선언하여 관리한다면, 가독성과 유지보수성에 도움을 줍니다. 다음은 예시이니 참고 부탁드립니다! 
```suggestion
        “private static final int MOVE_THRESHOLD = 4”
```